clients/privacy-center/components/ConsentItemCard.tsx

```diff
@@ -10,7 +10,7 @@ import {
   HStack,
 } from "@fidesui/react";
 import { ExternalLinkIcon } from "@chakra-ui/icons";
-import { ConsentItem } from "../types";
+import { ConsentItem } from "~/features/consent/types";
```

Comment:
Note that this should already be on `main` since I did a PR here: https://github.com/ethyca/fides/pull/1456